update_repo-template.sh

```diff
@@ -20,5 +20,5 @@ dir=$(dirname -- "$0") || exit
 . "$dir/update_repo.sh" "$1"
 
 if [ "${AUTOMATIC_UPLOAD-}" = 1 ]; then
-    "$dir/sync_qubes-os.org_repo.sh" "$1" $REPOS_TO_UPDATE
+    "$dir/sync_qubes-os.org_repo.sh" "$1" mirrors.list $REPOS_TO_UPDATE
```

Comment:
No, it's actually `update_repo.sh` that requires the argument, however strange that looks. But see the comment in main conversation.